deploy/addons/aliyun_mirror.json

```diff
@@ -0,0 +1,102 @@
+{
+  "kubernetesui/dashboard": "registry.cn-hangzhou.aliyuncs.com/google_containers/dashboard",
```

Comment:
thank you very much @denverdino  for this PR 

is there a way we could have this images synced by minikube team itself ? to ensure its integrity ?
ideally how about we have a Service Account add to minikube project and we can add a Github Acction that runs Sync every day and copies the addons images we have to alibaba

Where does currenly the Sync happen ? who runs it and how can we be sure the SHA of these images are same ?

Reply:
>> is there a way we could have this images synced by minikube team itself ? to ensure its integrity ?

It should be OK

>> Where does currenly the Sync happen ? who runs it and how can we be sure the SHA of these images are same ?

I run such synchronization everyday.  It is based on https://github.com/denverdino/image-syncer, it can sync the image repo for multi-arch images.

Reply:
@denverdino could you please provide me commands or scripts to run https://github.com/denverdino/image-syncer

so we could run that in minikube-ci ?


if we ship an image offically thorough minikube code I would like to ensure that image was published by automation and through open source workflow


how about adding it as a Cron Job in github actions simmilar to this job
https://github.com/kubernetes/minikube/blob/master/.github/workflows/time-to-k8s-public-chart.yml

in that job it runs every day and does a task and upload the result to a GCS bucket
we could add same job for Aliyun image sync that runs everyday.

what do you think about it?

Reply:
Sure, pls take a look for the actions

https://github.com/denverdino/image-syncer/blob/master/.github/workflows/go.yml

And let me know if it works